challengeFive.js

```diff
@@ -0,0 +1,46 @@
+// First attempt 
+
+let cardNumber = "";
+
+function hideCardNumber(number){
+    const cardNumberString = number.toString();
```

Comment:
Why is this called cardNumberString? It should be called cardNumber as the line of code converts the number to a string.